src/components/CodeView/index.tsx

```diff
@@ -81,11 +85,16 @@ type Props = PublicProps & DefaultProps & RouteComponentProps;
 export class CodeViewBase extends React.Component<Props> {
   static defaultProps: DefaultProps = {
     _scrollToSelectedLine: scrollToSelectedLine,
+    _sendPerfTiming: sendPerfTiming,
     _slowLoadingCharCount: SLOW_LOADING_CHAR_COUNT,
     _trimmedCharCount: TRIMMED_CHAR_COUNT,
     enableCommenting: process.env.REACT_APP_ENABLE_COMMENTING === 'true',
   };
 
+  onRenderCallback = (id: string, phase: string, actualDuration: number) => {
```

Comment:
```suggestion
  onRenderProfiler = (id: string, phase: string, actualDuration: number) => {
```

Reply:
We should likely add a link to the docs: https://github.com/reactjs/rfcs/blob/master/text/0051-profiler.md above this method.